themes/openy_themes/openy_rose/templates/paragraph/paragraph--grid-content--default.html.twig

```diff
@@ -45,7 +45,7 @@ set classes = [
   view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
 ]
 %}
-{% set grid_style = content.field_prgf_grid_style|render|trim %}
+{% set grid_style = content.field_prgf_grid_style.0['#markup'] %}
```

Comment:
Wel.... this is something that we're fighting with. `content.field_prgf_grid_style.0['#markup']` looks ugly and may lead to unpredictable errors with the Drupal core API changes.

We should be able to render only value, because it was cleaner up in template https://github.com/ymcatwincities/openy/blob/8.x-1.x/themes/openy_themes/openy_rose/templates/field/field--paragraph--field-prgf-grid-style--grid-content.html.twig#L42.